kernelci/config/lab.py

```diff
@@ -75,11 +79,18 @@ def from_yaml(cls, lab, kw):
         return cls(**kw)
 
 
+class Lab_SHELL(Lab):
```

Comment:
Not sure if you want to see us add the Lab_SHELL in a separate commit.

Reply:
Yes, the new shell class for lab should be its own patch  IMO.

Reply:
Alright, I split it off into a separate patch.

Reply:
Oh whoops, I might have put them in the wrong order...

Reply:
Nope, Github just shows them in reverse order to what I expect. We're good.